st2client/st2client/commands/pack.py

```diff
@@ -190,12 +190,41 @@ def __init__(self, resource, *args, **kwargs):
                                  help='Force pack installation.')
 
     def run(self, args, **kwargs):
+        if args.packs:
+            if len(args.packs) == 1:
+                args.pack = args.packs[0]
+                pack_info = self.manager.search(args, **kwargs)
+                content = getattr(pack_info, 'content', {})
+                if content:
+                    print '\nFor "%s" pack following content will be registered:\n' % args.pack
+                    for entity in content.keys():
+                        if entity != 'tests':
+                            print '%s:  %s' % (entity, content[entity]['count'])
+                    print '\nIt may take a while based on number of items in content.'
+            else:
+                pack_content = {}
+                for pack in args.packs:
```

Comment:
This is probably not problematic right now, but in case it becomes common that users install many packs at once they it might make sense to add filtering based on multiple pace names to the packs search endpoint (e.g. `?packs=pack1,pack2,pack3`).

This way we can avoid multiple HTTP requests and only need one.